core/src/main/groovy/noe/tomcat/configure/ConnectorSSLHostConfigTomcat.groovy

```diff
@@ -0,0 +1,109 @@
+package noe.tomcat.configure
+
+/**
+ * Abstraction for SSLHostConfig element of either Secure or Non-Secure Connector in Tomcat server.xml.
+ * It is used for transfer data from user to `TomcatConfigurator`.
+ * Provides default values if needed.
+ * *
+ * IMPORTANT
+ * <ul>
+ *   <li>Not all SSLHostConfig attributes are supported. Only the most used ones.</li>
+ *   <li>It is user responsibility to set values semantically, no validation is performed.</li>
+ * <ul>
+ *
+ * @link https://tomcat.apache.org/tomcat-9.0-doc/config/http.html#SSL_Support_-_SSLHostConfig
+ */
+class ConnectorSSLHostConfigTomcat {
```

Comment:
Should be also handled element `Certificate` to make this api change? 
https://tomcat.apache.org/tomcat-9.0-doc/ssl-howto.html

Not sure, just asking.

Reply:
Didnt understand your point

Reply:
There is possible configuraiton like: 
````
<Connector
    port="8443"
    protocol="org.apache.coyote.http11.Http11AprProtocol"
    secure="true"
    scheme="https"
    SSLEnabled="true"
  <SSLHostConfig
      caCertificateFile="/path/to/ca.pem"
      certificateVerification="require"
      certificateVerificationDepth="10" >
    <Certificate
        certificateFile="/path/to/ocsp-cert.crt"
        certificateKeyFile="/path/to/ocsp-cert.key" />
  </SSLHostConfig>
````

I am curious whether is possible to skip the element Certificate for now, and we will have implemented features addressing our needs.

Reply:
Ah, you mean that as there two elements are related together, it would be better to launch them together to this PR right ?

Reply:
Honestly, I don't know the details. If the use-case we have does not need this element I would consider skipping it ATM,, if it is needed, let's implement it.

Reply:
I implemented it and is pushed in order to take a look :)